temporian/test/api_test.py

```diff
@@ -71,6 +71,79 @@ def test_serialization(self):
         self.assertSetEqual(set(inputs.keys()), {"a"})
         self.assertSetEqual(set(outputs.keys()), {"b"})
 
+    def test_serialization_v2(self):
```

Comment:
Nit: some more descriptive names or docstrings in these tests would be nice 😅

Reply:
Done :)